pkg/wwan/mmagent/mmdbus/client.go

```diff
@@ -825,6 +816,28 @@ func (c *Client) getModemPhysAddrs(modemObj dbus.BusObject) (
 	return addrs, proto, nil
 }
 
+// Given device path from /sys, such as: /sys/devices/pci0000:00/0000:00:1c.0/0000:2c:00.0/mhi0/wwan/wwan0
+// it returns the address of the device on the given bus (e.g. "0000:2c:00.0" for "pci").
```

Comment:
Might make sense to add a comment that if the device is not on the particular bus, it returns an empty string (and not an error).

Reply:
Done